OpenDungeons/OpenDungeons
# Make it possible to run the game out of the source dir

Create symbolic links to the game resources on Linux when building out of the CMAKE_SOURCE_DIR (fixes #121)
Also replace CMAKE_CURRENT_BINARY_DIR by CMAKE_BINARY_DIR (we have only one CMakeLists.txt)

To test:

```
mkdir build && cd build
cmake ..
./opendungeons
```

This failed previously because of missing resources, it should work now.
